File: c#/types/strings/StringLength.cs
class StringLength {
  public static void Main(string[] argv) {
    string word = "hello";

    System.Console.WriteLine(string.Format("word = {0}", word));
    System.Console.WriteLine(string.Format("length: {0}\n", word.Length));

    string cite = "Stat rosa pristina nomine, nomina nuda tenemus.";
    System.Console.WriteLine(string.Format("cite = {0}", cite));
    System.Console.WriteLine(string.Format("length: {0}\n", cite.Length));
  }
}

File: java/types/strings/StringLength.java
class StringLength {
  public static void main(String[] argv) {
    String word = "hello";
    System.out.println(String.format("word = %s", word));
    System.out.println(String.format("length: %d\n", word.length()));

    String cite = "Stat rosa pristina nomine, nomina nuda tenemus.";
    System.out.println(String.format("cite = %s", cite));
    System.out.println(String.format("length: %d\n", cite.length()));
  }
}
